### DISCUSSION TITLE
Missing Option to set Topology View Filter Defaults to all Users registered

### DISCUSSION THREAD

**Discussion by JanCBredow** [mattieserver/netbox-topology-views#589] [Q&A]
### NetBox version

NetBox Community v4.1.4 (2024-10-15)

### Topology Views version

v4.1.0

### Feature type

New functionality

### Proposed functionality

Maybe i am a bit confused, but is there a way to set a Set of Filterrules to defaultvalues so it applies to every user opening the topology view?

<img width="1366" alt="Bildschirmfoto 2024-12-04 um 12 47 57" src="https://github.com/user-attachments/assets/c2303b55-5225-45a4-b514-a9ff76382961">

I want to set all these Filters to every user opening the view

### Use case

get control about the overview, set defaults for coworkers so they don't have to mess around with filters and stuff

### External dependencies

none i guess

**Comment by dreng** [comment]
> Maybe i am a bit confused, but is there a way to set a Set of Filterrules to defaultvalues so it applies to every user opening the topology view?

Yes and no. There are no default filters. This is the normal NetBox behaviour and has got nothing to do with the plugin. That said, you can use Saved Filters. They do not set a default but they are a convenient way to get to default quite quickly. Saved Filters are a standard NetBox feature. In your screenshot, just click the Save button. You may give it a name like "Default". You can reuse that Saved Filter in the filter tab then.

Converting this question into a discussion, since there's nothing to do for us here.
